set.mm

```diff
@@ -62783,17 +62783,34 @@ ordered pairs (for use in defining operations).  This is a special case
 
   ${
     fvoveq1d.1 $e |- ( ph -> A = B ) $.
-    $( Equality deduction for nested function and operation value.
+    $( Equality deduction for nested function and operation value.  Technical
+       theorem to be used to reduce the size of a significant amount of proofs.
        (Contributed by AV, 23-Jul-2022.) $)
     fvoveq1d $p |- ( ph -> ( F ` ( A O C ) ) = ( F ` ( B O C ) ) ) $=
```

Comment:
As mentioned in the PR that introduced it, I think this should be relabeled fvoveq2d (and same with fvoveq2) since it is the equality theorem when the **second** variable in the expression ``( F ` ( A O C ) )`` is substituted for an equal one.
A global text search&replace would do it (with a space preceding the word, i.e., " fvoveq1d" --> " fvoveq2d").

Reply:
I think this is a matter of opinion: it is the second variable in the whole expression, but it is the first variable in the operation (you could read "fvoveq1d" as "fv(oveq1)d". What do others think about this?

Reply:
Ok. Both are justifiable, and that's not that important. I'm fine with both.